.github/ISSUE_TEMPLATE/enhancement-request.yaml

```diff
@@ -0,0 +1,89 @@
+name: "Security docs enhancement request"
+description: Submit an issue to the security documentation team to request an improvement to the Elastic Security documentation.
+title: "[enhancement]: "
+labels: "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Use this form to submit an enhancement request to the [@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs) team. 
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Summarize your request here. What is the problem?
+      placeholder: |
+        What: We need documentation for X.
+        Why: We've received support requests that indicate customers are struggling with Y in versions A and B. 
+    validations:
+      required: true
+  - type: textarea
+    id: related
+    attributes:
+      label: Related links / assets
+      description: How can we learn more about the problem?
+      value: |
+        Please include each of the following, if applicable:
+        Subject matter expert:
+        Figma link(s):
+        Github epic link(s):
+        Github issue link(s):
+    validations:
+      required: false
+  - type: dropdown
+    id: doc-set
+    attributes:
+      label: Which documentation set needs improvement?
+      description: Stateful, Serverless, or both?
+      options:
+        - Stateful and Serverless
+        - Stateful only
+        - Serverless only
+        - Unknown / Not applicable
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: doc-set-differences
+    attributes:
+      label: Feature differences
+      description: If you selected both Stateful and Serverless above, please describe how, if at all, the feature differs in each deployment method.
+      placeholder: The feature is identical in Stateful and Serverless.
+    validations:
+      required: true
```

Comment:
I think we can nix this section since this is an enhancement request. We can determine if we need to apply it to both doc sets. 
```suggestion

```